heroic-component/src/main/java/com/spotify/heroic/metric/QueryMetricsResponse.java

```diff
@@ -84,6 +88,8 @@ public void serialize(
             g.writeFieldName("result");
             serializeResult(g, common, result);
 
+            g.writeObjectField("rawSampleSize", response.getPreAggregationSampleSize());
```

Comment:
True, it was a bit confusing. I was trying to avoid the long preAggregationSampleSize, but now I've pushed an update that uses that name.